src/main/java/io/pakland/mdas/githubstats/infrastructure/shell/configuration/ShellConfiguration.java

```diff
@@ -1,28 +1,40 @@
 package io.pakland.mdas.githubstats.infrastructure.shell.configuration;
 
-import io.pakland.mdas.githubstats.infrastructure.shell.components.ShellTeamComponent;
-import io.pakland.mdas.githubstats.infrastructure.shell.components.ShellUserComponent;
+import io.pakland.mdas.githubstats.infrastructure.shell.controller.ShellController;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.shell.command.CommandRegistration;
 
 @Configuration
-public class CommandConfiguration {
+public class ShellConfiguration {
+
+    private final ShellController shellController;
+
+    public ShellConfiguration(ShellController shellController) {
+        this.shellController = shellController;
+    }
 
     @Bean
     public CommandRegistration buildUserCommand() {
-        ShellUserComponent shellUserComponent = new ShellUserComponent();
 
         return CommandRegistration.builder()
-                .command("user")
-                .description("Get data from a specified user.")
+                .command("get")
```

Comment:
We have taken a different approach in #193 which we strongly believe it is more understandable and clean. I would consider keeping the code in #193 and removing what has been changed here.

Reply:
Allright